entry_tasks/16_pick_movie_by_genre_or_actor/16_pick_movie_by_genre_or_actor.py

```diff
@@ -0,0 +1,47 @@
+if __name__ == '__main__':
+    genres = {
+        'comedy': ['Meet the Parents', 'Anger Management'],
+        'adventures': ['Mummy'],
+        'romantic': ['Vanilla Sky', 'Meet Joe Black'],
+        'drama': ['Meet Joe Black'],
+        'thriller': ['Vanilla Sky'],
+        'action': ['Mission Impossible']
+    }
+
+    actors = {
+        'Robert De Niro': ['Meet the Parents'],
+        'Ben Stiller': ['Meet the Parents'],
+        'Adam Sandler': ['Anger Management'],
+        'Jack Nicholson': ['Anger Management'],
+        'Brendan Fraser': ['Mummy'],
+        'Rachel Weisz': ['Mummy'],
+        'Tom Cruise': ['Vanilla Sky', 'Mission Impossible'],
+        'Penelope Cruz': ['Vanilla Sky'],
+        'Cameron Diaz': ['Vanilla Sky'],
+        'Brad Pitt': ['Meet Joe Black'],
+        'Anthony Hopkins': ['Meet Joe Black'],
+        'Jeremy Renner': ['Mission Impossible']
+    }
+
+    search_by_genre = input('Search by Genre (y/n): ')
+    if search_by_genre == 'y':
+        print("Available Genres:", list(genres.keys()))
+        genre_choice = input('Enter genre: ')
+        for i in genres.keys():
+            if i == genre_choice:
+                print("Available Movies:", genres.get(i))
```

Comment:
So if you know about genres.get(i), why do you loop over all keys and then pick it if got a match.

Instead:
```py
for i in genres.keys():
    if i == genre_choice:
        print("Available Movies:", genres.get(i))
```
do
```py
genre_selected = genres.get(genre_choice)
if genre_selected is None:
    print('invalid input')
else:
    "your logic here"
```